src/amo/components/Search/index.js

```diff
@@ -120,10 +124,19 @@ export class SearchBase extends React.Component {
       <SearchSort filters={filters} pathname={pathname} />
     ) : null;
 
+    if (!queryParams.q) {
```

Comment:
This should be using `filters`, not `queryParams`. It's a bit of a style/convention difference but we use `filters` everywhere internally as they are better-labelled. For instance, you can use `filters.query` here instead of `queryParams.q`, which is more readable and informative.

Additionally, this should account for an empty search (eg `''`), so the check should actually be `if (!filters.query || filters.query.length === 0) {`